### ISSUE TITLE
S3 to GCS functionality

### ISSUE THREAD

**Issue by elyse-weiss** [move-coop/parsons#516]
I have a use case where I would like to move files directly between S3 and GCS. @tonywhittaker offered the following resources that could be helpful:

https://cloud.google.com/storage/docs/interoperability#using_the_gsutil_command_line
https://github.com/GoogleCloudPlatform/gsutil/

I would love it if someone in the community could take a stab at adding this to Parsons!

**Comment by shaunagm** [comment]
Yeah, this seems like it could be part of the effort to add more utilities in #554
